pyproject.toml

```diff
@@ -5,3 +5,20 @@ requires = [
 ]
 
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+xfail_strict = true
+addopts = [
+    "-ra",
+    "--cov=skhep",
+    "--cov-branch",
```

Comment:
Coverage is 86% which is lower than the coveralls reported 89% as branching is also taken into account (I think at least!).